lecturework.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Lesson 8 - Loops</title>
+  </head>
+
+  <body>
+    <div>
+      <h3>List of Products</h3>
+      <div>
+        <span>Iphone</span><br />
+        <span>Price: 900 €</span>
+        <input
+          type="button"
+          id="addPhoneToCart"
+          value="Add to shopping cart"
+        />
+      </div>
+      <br />
+      <div>
+        <span>Nintendo DS</span><br />
+        <span>Price: 1200 €</span>
+        <input
+          type="button"
+          id="addNintendoToCart"
+          value="Add to shopping car"
+        />
+      </div>
+      <br />
+      <div>
+        <span>MacBook Air</span><br />
+        <span>Price: 3200 €</span>
+        <input
+          type="button"
+          id="addMacbookToCart"
+          value="Add to shopping car"
+        />
+      </div>
+    </div>
+    <hr />
+    <div>
+      <h3>Shopping Cart</h3>
+      <input
+        type="button"
+        id="showShoppingCart"
+        value="Show Shopping Cart Products"
+      />
+      <div id="shoppingCart">
+        <!-- Display shopping cart products-->
+      </div>
+      <h3>
+        Total Price
+      </h3>
+
+      <span id="total">
+        <!-- show total price-->
+      </span>
+      <input type="button" id="totalPrice" value="Show total price" />
+    </div>
+
+    <script>
+
+      const phoneToCartEl = document.getElementById("addPhoneToCart");
+      const nintendoToCartEl = document.getElementById("addNintendoToCart");
+      const macbookToCartEl = document.getElementById("addMacbookToCart");
+      const showShoppingCartEl = document.getElementById("showShoppingCart");
+      const totalPriceEl = document.getElementById("total");
+
```

Comment:
this is throwing an issue because you have not defined shoppingCart array before start using the variable and add your products.
you need to do this here:
`const shoppingCart = [];`

Reply:
Thank you!